library/src/webapp/js/portal/portal.all.sites.js

```diff
@@ -59,7 +42,7 @@ portal.allsites.setAllOrNoneStarStates = function () {
 
   sidebar.querySelectorAll(".favorites-select-all-none").forEach(function (selectAllNone) {
 
-    const termContainer = selectAllNone.closest(".fav-sites-term");
+    const termContainer = selectAllNone.closest(".card");
 
```

Comment:
card is also a bootstrap class.

Reply:
I know. Tell me why this is a bad idea. Explain it - I'm interested in your reasoning.

Reply:
Do you think manipulating a Bootstrap native class is a good idea? It's not a good practice and it can have an effect on some other parts where we are using it. Even if you are sure it won't have any effect elsewhere. I still don't think it's a good idea. It's best if we create a custom class and then modify it.

Reply:
all these changes are scoped though, as long as there is a card above the select all button, it should be fine. I understand your reasoning though, it is good reasoning.

Reply:
Done